CodeToReview.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+   O | O | O 
+  -----------
+   O | O | O 
+  -----------
+   O | O | O 
+     
+ */
+package tictactoe;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    /**
+     * @param args the command line arguments
+     * @throws java.lang.InterruptedException
+     */
+    public static void main(String[] args) throws InterruptedException {
+        Scanner input = new Scanner(System.in);
+        String a1 = "1";
+        String a2 = "2";
+        String a3 = "3";
+        String a4 = "4";
+        String a5 = "5";
+        String a6 = "6";
+        String a7 = "7";
+        String a8 = "8";
+        String a9 = "9";
+        int inO;
+        int inX = 0;
+        boolean winner = false;
+        PrintBoard(a1,a2,a3,a4,a5,a6,a7,a8,a9);
+        System.out.printf("How to play - wait for your turn and when it comes write the number of the field that you want to draw in (from 1 to 9)\n");
+        System.out.printf("Player O's turn: ");
+        inO = input.nextInt();
+        while (true) {
+            switch (inO) {
+                case 1:
+                    if (a1 == "1") {
+                        a1 = "O";
+                    }
+                    break;
+                case 2:
+                    if (a2 == "2") {
+                    a2 = "O";
+                    }
+                    break;
+                case 3:
+                    if (a3 == "3") {
+                    a3 = "O";
+                    }
+                    break;
+                case 4:
+                    if (a4 == "4") {
+                    a4 = "O";
+                    }
+                    break;
+                case 5:
+                    if (a5 == "5") {
+                    a5 = "O";
+                    }
+                    break;
+                case 6:
+                    if (a6 == "6") {
+                    a6 = "O";
+                    }
+                    break;
+                case 7:
+                    if (a7 == "7") {
+                    a7 = "O";
+                    }
+                    break;
+                case 8:
+                    if (a8 == "8") {
+                    a8 = "O";                        
+                    }
+                    break;
+                case 9:
+                    if (a9 == "9") {
+                    a9 = "O";
+                    }
+                    break;
+                default:
+                    System.out.printf("An error occured. Please accept it politely and restart the game.");
+                    break;
+            }
+            System.out.printf("\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n");
```

Comment:
Might be better to use the repeater syntax. It will become easier to know how many next line chars are used.